UNRELEASED.md

```diff
@@ -6,6 +6,8 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 ### Breaking changes
 
+- Providing `Card.Section` title with a `JSX Element` will no longer wrap the title in a `Subheading` component. ([#781](https://github.com/Shopify/polaris-react/pull/781))
```

Comment:
We would not consider this a breaking change, as our markup is not part of our API. You can delete this entry.